lib/server/auth.ts

```diff
@@ -1,23 +1,31 @@
-import jwt, { Secret } from 'jsonwebtoken';
+import { jwtVerify, JWTPayload } from 'jose';
 
-export function getJwtSecretKey(): Secret {
-    // Secret is hardcoded for now
-    // TODO: Need to figure out where the secret will be stored
-    const secret = Buffer.from('Zn8Q5tyZ/G1MHltc4F/gTkVJMlrbKiZt', 'base64');
+/**
```

Comment:
Changed to using 'jose' package to verify the token. This is because the "Edge runtime environment does not include the 'crypto' Node.js core module, and jsonwebtoken relies on this module for cryptographic operations.

The 'jose' package is designed to work without this, using standard APIs like Web Cryptography API which is supported in browsers.